src/js/hooks/cycleCount/useCountStepTable.jsx

```diff
@@ -141,12 +141,16 @@ const useCountStepTable = ({
       // When the input is blurred, we'll call the table meta's updateData function
       const onBlur = () => {
         if (isEdited) {
-          table.options.meta?.updateData(cycleCountId, original.id, id, value);
+          if (columnPath === cycleCountColumn.QUANTITY_COUNTED) {
+            setValue(parseInt(value, 10) || 0);
+            table.options.meta?.updateData(cycleCountId, original.id, id, parseInt(value, 10) || 0);
```

Comment:
1. You can assign the result of parseInt to a variable
2. I do not tolerate using else :imp: